scripts/config/createJestConfig.js

```diff
@@ -45,7 +45,8 @@ module.exports = (resolve, rootDir) => {
     "testResultsProcessor",
     "transform",
     "transformIgnorePatterns",
-    "watchPathIgnorePatterns"
+    "watchPathIgnorePatterns",
+    "moduleNameMapper"
```

Comment:
this will allow us to add our aliases to our package.json in a format jest understands (regex)

```
"jest": {
    "moduleNameMapper": {
      "libs(.*)$": "<rootDir>/libs/$1"
    }
  }
```